src/index.js

```diff
@@ -135,6 +134,7 @@ class DockAPI {
 
     await this.initKeyring(keyring);
 
+    this.anchorModule = new AnchorModule();
```

Comment:
instead of creating a new anchormodule instance each connect, would be better to simply remove this line https://github.com/docknetwork/sdk/blob/28f24976e53534a4477a3b627d67e9ffa6a3f8b8/src/index.js#L165 so that it isnt deleted.

good spot either way though

Reply:
Won't be the problem that `anchorModule` will have disconnected `this.api` under the hood?
On `dock.anchor` getter call `AnchorModule` with disconnected API will be returned (https://github.com/docknetwork/sdk/blob/master/src/index.js#L288).

By the way, I'm not sure what's the problem with recreating `AnchorModule`? This class doesn't have a constructor and it doesn't extend any other class.

Reply:
the disconnected API will be returned yes, but there are parts of the anchor module that do not require an api connection to be able to use. thats why anchor module is constructed outside of init unlike the others (so its always available)

if you call `dockapi.init()` again after disconnect, it will call `setApi` in anchor module so that the latest api connection

Reply:
I'd say it's better to either move these methods to some module without API dependency or make them static, but anyway I changed behavior to keep `anchorModule` always initiated.

Reply:
i agree, hopefully we can get some refactoring time here. thats quite a bit we had planned but SDK was put on into maintenance mode so we can focus on other projects